### ISSUE TITLE
Add Current Application Mappings

### ISSUE THREAD

**Issue by jateeter** [LBNL-ETA/BEDES-Manager#50]
Import the following mappings into BEDES-Manager.lbl.gov:

1 - Energy Start Portfolio Manager: 
2 - BuildingSynch
3 - HPXML

**Comment by TravisWalterLBNL** [comment]
Closing because it's not really in the scope of mapping manager functionality.

**Comment by TravisWalterLBNL** [comment]
Reopening because it's not a mapping manager fix, but we don't want to forget to do it, so will just change assignee.

**Comment by robHitchcock** [comment]
I have just noticed that Data Type and Unit of Measure do not seem to be retained from my previous NYSERDA application mapping import. We plan to clean the database prior to deploying the new site. We should verify correct import when we populate the new site.

**Comment by TravisWalterLBNL** [comment]
@robHitchcock Does issue #178 capture the problem you described in your last comment, or is there something else?

**Comment by TravisWalterLBNL** [comment]
To complete this task, we will first need to convert several mappings from BEDES v2.3 to v2.4.

**Comment by robHitchcock** [comment]
@TravisWalterLBNL I believe my comment here about losing Data Type and Unit of Measure must refer to the display of a mapping under Applications rather than Composite Terms (issue #178) since Data Type is not displayed under Composite Terms.
I think it is confusing to only display Data Type and Unit on the left side (Application Term) of this page. I think Data Type and Unit should also be displayed for the Mapped Term on the right side since they might be different.
![image](https://user-images.githubusercontent.com/7998768/117353907-96ced780-ae65-11eb-9a80-ab3f21fefc8e.png)

**Comment by TravisWalterLBNL** [comment]
@robHitchcock I agree that the details should be displayed for both the application term and the mapped term. I will create a separate issue.

**Comment by TravisWalterLBNL** [comment]
Closing because all of the currently completed application mappings have been uploaded and made public (even if they haven't been marked as approved).
